assets/javascripts/application.js

```diff
@@ -45,7 +45,8 @@ var sso = require('./modules/sso.js'),
     charCounter = require('./modules/charCounter.js'),
     addRemove = require('./modules/addRemove.js'),
     modalDialog = require('./modules/modalDialog.js'),
-    dynamicGaTags = require('./modules/dynamicGaTags.js');
+    dynamicGaTags = require('./modules/dynamicGaTags.js'),
+    listCollapse = require('./modules/listCollapse.js');
```

Comment:
Perhaps this should be named `resonsiveNav` so as not to encourage collapsing lists all over the place